static_report/src/App.tsx

```diff
@@ -97,7 +98,19 @@ function AppComparison() {
   );
 }
 
+function AppIndexReport() {
+  return (
+    <Suspense fallback={<Loading />}>
+      <IndexReport data={window.PIPERIDER_INDEX_REPORT_DATA} />
```

Comment:
you probably don't need this prop this high up in the component. (see below comment)